### DISCUSSION TITLE
Interpolation in Project Templates

### DISCUSSION THREAD

**Discussion by SalvadorCali** [mia-platform/community#92] [Q&A] [answered]
Hi everyone!
We want to add a new dashboard to the default template of our projects inside the **Project Templates** section of the CMS. 
Which is the right way to interpolate in order to set the correct value for CLUSTER and NAMESPACE? 
This is an example url of the dashboard:
`https://grafana.mia-platform.eu/d/testId/project-logs?orgId=15&var-cluster=[CLUSTER]&var-namespace=[NAMESPACE]&var-container=All&theme=light&kiosk=tv`

**Comment by davidebianchi** [comment, marked as answer]
Hi @SalvadorCali! Have you already seen [the archive docs](https://docs.mia-platform.eu/docs/development_suite/set-up-infrastructure/create-project#how-to-create-a-project-archive)?
